src/fi_tostr.c

```diff
@@ -487,6 +487,14 @@ static void fi_tostr_atomic_op(char *buf, enum fi_op op)
 	}
 }
 
+static void fi_tostr_version(char *buf, uint32_t op)
+{
+	/* PACKAGE_VERSION is defined in config.h by configure */
+	static const char version_str[] = PACKAGE_VERSION;
+	(void)op;
+	strcatf(buf, version_str);
```

Comment:
Touche.  Normally I do that to ensure that only one copy of the string constant gets created, but since anything else that wants PACKAGE_VERSION will call this, your suggestion makes sense.  I will change that in the next version.